spec/mailers/previews/digest_mailer_preview.rb

```diff
@@ -30,7 +30,7 @@ class DigestMailerPreview < ActionMailer::Preview
   # Preview emails at http://localhost:3000/rails/mailers/digest_mailer
 
   def work_packages
-    notifications = Notification.where(resource_type: "WorkPackage")
+    notifications = Notification.where(resource_type: "WorkPackage").limit(10)
```

Comment:
without this line, on a larger instance, it will output all notifications in one mail